ui-admin/src/study/participants/survey/EnrolleeSurveyView.tsx

```diff
@@ -41,13 +42,20 @@ export function RawEnrolleeSurveyView({ enrollee, configSurvey, responses }:
     return <div>Most recent response has no data yet </div>
   }
 
+  const answerVersions = _uniq(lastResponse.answers.map(ans => ans.surveyVersion))
+  const versionString = `version${answerVersions.length > 1 ? 's' : ''} ${answerVersions.join(', ')}`
```

Comment:
terra-ui uses [pluralize](https://www.npmjs.com/package/pluralize) for strings like this. It's obviously trivial to implement it each time, but I'm wondering if we should bring that package in since this is a very common thing to need to do and it's a nice utility to have